### ISSUE TITLE
PAR can't reproduce a smooth time series

### ISSUE THREAD

**Issue by arsinnius** [sdv-dev/SDV#523]
### Environment details

The code was run on Colab
* SDV version: sdv-0.11.0
* Python version: 3.7.11
* Operating System: posix / Linux / Release 5.4.104+

### Problem description

I'm using PAR to model macro variables. First, I modeled the VIX volatility index with no apparent problem. Next, I tried gross domestic product. The original GDP is plotted in the first figure - a smooth curve. The sample is plotted in the second figure. Something is clearly wrong. This raises doubt about the validity of the VIX sample.

![image](https://user-images.githubusercontent.com/84858555/126768764-affa3f54-2105-40e6-95c1-002ab9d5bba1.png)
![image](https://user-images.githubusercontent.com/84858555/126768975-13eb8fd7-e94c-4742-8647-6a2c5df97c56.png)

The data was downloaded from the FRED website as a csv file and converted to a pandas df. The df has two columns - DATE and GDP.

### What I already tried

I tried a second time and got the following:
![image](https://user-images.githubusercontent.com/84858555/126769504-22b9b319-f942-4f5f-b0e8-23ea639a45e3.png)

Here is the code I ran:
```
date_cols =['DATE'] 
gdp_df = pd.read_csv('./csv_data/GDP.csv', parse_dates=date_cols)
gdp_df.DATE = pd.to_datetime(gdp_df.DATE)
gdp_df.plot(x="DATE")

sequence_index = 'DATE'
model = PAR(sequence_index=sequence_index)
model.fit(gdp_df)
gdp_syn_1 = model.sample()
gdp_syn_1.plot(x='DATE')
```

**Comment by npatki** [comment]
Thanks for the detailed explanation and feedback @arsinnius. I am able to get some improvements when increasing epochs. Unfortunately, it never ends up looking like a smooth curve. Seems like it's difficult to for the neural network to learn.

I'm curious about how you're planning to use the synthetic data? A key strength of the PAR model is being able to model multi-sequence data. Since you only have a single sequence, I'm wondering if it could make sense to model this particular data as tabular and sort the synthetic values by `DATE` afterwards.

Eg.

```python
from sdv.tabular import GaussianCopula

model = GaussianCopula(default_distribution='beta')
model.fit(gdp_df)
synthetic_data = model.sample(num_rows=289)
```

This got me more relative accuracy, though the curve still wasn't smooth.

<img width="1097" alt="image" src="https://user-images.githubusercontent.com/3227804/176791793-bddfcebe-bb52-49b5-b0c8-a4dd8f6f8de5.png">

**Comment by npatki** [comment]
Hello, I'm turning this into a feature request and slightly renaming it. We'll keep this open to track as we make progress.
